### ISSUE TITLE
Error with scoreboard module

### ISSUE THREAD

**Issue by Oruss7** [slipcor/pvparena#164]
My friend have an error with the scoreboard module. 

He have no stats, no color and a nice error when leave the arena :

```
[Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'pa' in plugin pvparena v1.3.2.86
at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46) ~[spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141) ~[spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
at org.bukkit.craftbukkit.v1_8_R3.CraftServer.dispatchCommand(CraftServer.java:641) ~[spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
at net.minecraft.server.v1_8_R3.PlayerConnection.handleCommand(PlayerConnection.java:1162) [spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
at net.minecraft.server.v1_8_R3.PlayerConnection.a(PlayerConnection.java:997) [spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
at net.minecraft.server.v1_8_R3.PacketPlayInChat.a(PacketPlayInChat.java:45) [spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
at net.minecraft.server.v1_8_R3.PacketPlayInChat.a(PacketPlayInChat.java:1) [spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
at net.minecraft.server.v1_8_R3.PlayerConnectionUtils$1.run(SourceFile:13) [spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_60]
at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_60]
at net.minecraft.server.v1_8_R3.SystemUtils.a(SourceFile:44) [spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
at net.minecraft.server.v1_8_R3.MinecraftServer.B(MinecraftServer.java:715) [spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
at net.minecraft.server.v1_8_R3.DedicatedServer.B(DedicatedServer.java:374) [spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
at net.minecraft.server.v1_8_R3.MinecraftServer.A(MinecraftServer.java:654) [spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:557) [spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
at java.lang.Thread.run(Thread.java:745) [?:1.8.0_60]
Caused by: java.lang.NullPointerException
at net.slipcor.pvparena.modules.scoreboards.ScoreBoards.remove(ScoreBoards.java:137) ~[?:?]
at net.slipcor.pvparena.modules.scoreboards.ScoreBoards.resetPlayer(ScoreBoards.java:91) ~[?:?]
at net.slipcor.pvparena.loadables.ArenaModuleManager.resetPlayer(ArenaModuleManager.java:254) ~[?:?]
at net.slipcor.pvparena.arena.Arena.resetPlayer(Arena.java:1285) ~[?:?]
at net.slipcor.pvparena.arena.Arena.removePlayer(Arena.java:1050) ~[?:?]
at net.slipcor.pvparena.arena.Arena.playerLeave(Arena.java:865) ~[?:?]
at net.slipcor.pvparena.commands.PAG_Leave.commit(PAG_Leave.java:54) ~[?:?]
at net.slipcor.pvparena.PVPArena.onCommand(PVPArena.java:378) ~[?:?]
at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44) ~[spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
... 15 more
```

**Comment by slipcor** [comment]
what versions are the modules?

/pa version

**Comment by Oruss7** [comment]
1.3.2.51

**Comment by slipcor** [comment]
Well I cannot explain that. Did they try to restart the server? If they have an error on player removal there has to be an error on player adding, too - the scoreboard module is not correctly loaded, I assue they did a reload or install without proper restart

**Comment by Oruss7** [comment]
When we enter to the lounge, scoreboard doesn't load correctly

**Comment by Oruss7** [comment]
Of course, we have stop server, remove modules and reinstall

**Comment by slipcor** [comment]
can you try version build 71 of https://ci2.craftyn.com/job/PVP%20Arena%20Modules/ ?

**Comment by Oruss7** [comment]
Error
````
[00:40:36] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'pa' in plugin pvparena v1.3.2.91
at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46) ~[spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141) ~[spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
at org.bukkit.craftbukkit.v1_8_R3.CraftServer.dispatchCommand(CraftServer.java:641) ~[spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
at net.minecraft.server.v1_8_R3.PlayerConnection.handleCommand(PlayerConnection.java:1162) [spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
at net.minecraft.server.v1_8_R3.PlayerConnection.a(PlayerConnection.java:997) [spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
at net.minecraft.server.v1_8_R3.PacketPlayInChat.a(PacketPlayInChat.java:45) [spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
at net.minecraft.server.v1_8_R3.PacketPlayInChat.a(PacketPlayInChat.java:1) [spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
at net.minecraft.server.v1_8_R3.PlayerConnectionUtils$1.run(SourceFile:13) [spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_60]
at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_60]
at net.minecraft.server.v1_8_R3.SystemUtils.a(SourceFile:44) [spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
at net.minecraft.server.v1_8_R3.MinecraftServer.B(MinecraftServer.java:715) [spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
at net.minecraft.server.v1_8_R3.DedicatedServer.B(DedicatedServer.java:374) [spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
at net.minecraft.server.v1_8_R3.MinecraftServer.A(MinecraftServer.java:654) [spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:557) [spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
at java.lang.Thread.run(Thread.java:745) [?:1.8.0_60]
Caused by: java.lang.NullPointerException
at net.slipcor.pvparena.modules.scoreboards.ScoreBoards.remove(ScoreBoards.java:137) ~[?:?]
at net.slipcor.pvparena.modules.scoreboards.ScoreBoards.resetPlayer(ScoreBoards.java:91) ~[?:?]
at net.slipcor.pvparena.loadables.ArenaModuleManager.resetPlayer(ArenaModuleManager.java:254) ~[?:?]
at net.slipcor.pvparena.arena.Arena.resetPlayer(Arena.java:1285) ~[?:?]
at net.slipcor.pvparena.arena.Arena.removePlayer(Arena.java:1050) ~[?:?]
at net.slipcor.pvparena.arena.Arena.playerLeave(Arena.java:865) ~[?:?]
at net.slipcor.pvparena.commands.PAG_Leave.commit(PAG_Leave.java:54) ~[?:?]
at net.slipcor.pvparena.PVPArena.onCommand(PVPArena.java:379) ~[?:?]
at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44) ~[spigot-1.8.8.jar:git-Spigot-db6de12-07c3001]
... 15 more
````

**Comment by slipcor** [comment]
I will try to fix this even though this should not be possible, unless you maybe had an issue on server start? There can not possibly be a NPE. I will add specific debug and warnings to be output to the server. Please check with latest modules being built right now!

https://ci2.craftyn.com/job/PVP%20Arena%20Modules/ #72

**Comment by Oruss7** [comment]
Not working: http://pastebin.com/2b6H9wgr

**Comment by slipcor** [comment]
You linked the config, not the error, @Oruss7  :)

**Comment by Oruss7** [comment]
Yeah, i known. I see any errors :confused:

**Comment by slipcor** [comment]
okay. that was the base of this ticket though sooo it is important to know that the error is gone BUT it does not work, and there is no new error ;)

**Comment by Oruss7** [comment]
Fix with latest spigot 1.8.8 (F*cking old version ^^).

And Nametag are colored with scoreboard ! Youpi
